lib/skills.go

```diff
@@ -0,0 +1,17 @@
+package lib
+
+// Skill contains a skill
+type Skill struct {
+	ID                    int     `json:"Id"`
+	Level                 int     `json:"Level"`
+	PercentageOfNextLevel float64 `json:"PercentageOfNextLevel"`
```

Comment:
What would you think of making this `PercentNextLevel` to be just a bit shorter.